Podfile

```diff
@@ -30,7 +30,7 @@ def wordpress_shared
     ##pod 'WordPressShared', :path => '../WordPress-iOS-Shared'
 
     ## while PR is in review:
-    ##pod 'WordPressShared', :git => 'https://github.com/wordpress-mobile/WordPress-iOS-Shared.git', :commit => '30809ffa12074b347355f73661d807766eade30b'
+    ##pod 'WordPressShared', :git => 'https://github.com/wordpress-mobile/WordPress-iOS-Shared.git', :commit => '4781c4764f69ca116c93c15a1297616b8920c4de'
```

Comment:
For this to work, I had to comment out line 27 (with `#`), and uncomment this line. Otherwise, the branch was unable to find the events added via the corresponding [`WordPressShared` PR](https://github.com/wordpress-mobile/WordPress-iOS-Shared/pull/179).

Once that PR has been merged, tagged, and published as a beta version, we'll flip it back prior to merging this PR.

Reply:
Thanks for that!

I released WordPress-iOS-Shared 1.7.0-beta.2 to pod trunk so it should be picked up automatically now 👍